packages/electron-app/README.md

```diff
@@ -11,3 +11,23 @@ This command assumes that we've built a CRA app inside `packages/light-apps`. It
 ### `yarn start`
 
 Runs `electron-webpack dev`, which tells Electron to load http://localhost:3000, which is most likely the port on which CRA's dev mode runs. It reloads Electron on any code change inside the electron codebase.
+
+## Electron Security Checklist:
+https://electronjs.org/docs/tutorial/security#1-only-load-secure-content
+
+Inspired by: https://github.com/paritytech/fether/pull/451
+
+#### Cheatsheet
+1. Only Load Secure Content (https/wss/ftps over http/ws/ftp)
+2. Disable Node.js Integration for Remote Content
+  * Prevents XSS attacks from escalating to a Remote Code Execution Attack - limits the attack surface to the renderer process.
+  <pre>
+  const mainWindow = new BrowserWindow({
+    webPreferences: {
+      nodeIntegration: false,
+      nodeIntegrationInWorker: false,
+      preload: path.join(app.getAppPath(), 'preload.js')
```

Comment:
Let's try to not have any preload script on light-ui. On fether the one crucial place where we needed main/renderer communication was the secure token, which we don't have here.

Reply:
noted.